routes/celebrities.routes.js

```diff
@@ -26,7 +26,16 @@ router.get('/',(req,res) => {
         .catch(err => console.log(err))
 })
 
+router.post('/:id/delete', (req, res, next) => {
+    const {id} = req.params
 
+    Celebrity
+      .findByIdAndDelete(id)
+      .then (deletedCelebrity => {
+        res.redirect(`/celebrities`)
+      })
+      .catch(err => console.log(err))
+  });
```

Comment:
inside the promise, if you are not using the variable - you do not need to write it:

```
      .then (()=> {
        res.redirect(`/celebrities`)
      })
```